src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
```

Comment:
Add a new line to the end of the file)

<img width="623" alt="Screenshot 2023-12-08 at 20 01 23" src="https://github.com/Andriy-Hrynevetskyi/spring-boot-book-store/assets/45499568/afbc7e7b-eb78-4a73-a85b-550a9e6b6959">